backend/ddang/src/test/java/com/ddang/ddang/region/infrastructure/api/RestTemplateInitRegionProcessorTest.java

```diff
@@ -0,0 +1,196 @@
+                Collections.singletonMap("accessToken", accessToken)
+        );
+
+        mockRestServiceServer
+                .expect(requestTo(matchesPattern(
+                        "https://sgisapi.kostat.go.kr/OpenAPI3/auth/authentication.json.*"
+                )))
+                .andRespond(withSuccess(
+                        objectMapper.writeValueAsString(accessTokenResponse),
+                        MediaType.APPLICATION_JSON
+                ));
+
+        final String firstRegionUri = "https://sgisapi.kostat.go.kr/OpenAPI3/addr/stage.json?" +
+                "accessToken=" +
+                accessToken;
+        final ResultApiRegionResponse firstResultApiRegionResponse = new ResultApiRegionResponse(
+                "서울특별시",
+                "11"
+        );
+        final TotalApiRegionResponse openApiResultApiRegionResponse = new TotalApiRegionResponse(
+                List.of(firstResultApiRegionResponse)
+        );
+
+        mockRestServiceServer
+                .expect(requestTo(firstRegionUri))
+                .andRespond(withSuccess(
+                        objectMapper.writeValueAsString(openApiResultApiRegionResponse),
+                        MediaType.APPLICATION_JSON
+                ));
+
+        final String secondRegionCode = "11";
+        final String secondRegionUri = "https://sgisapi.kostat.go.kr/OpenAPI3/addr/stage.json?" +
+                "accessToken=" +
+                accessToken +
+                "&cd=" +
+                secondRegionCode;
+        final ResultApiRegionResponse secondResultApiRegionResponse = new ResultApiRegionResponse(
+                "강남구",
+                "11230"
+        );
+        final TotalApiRegionResponse secondOpenApiResultApiRegionResponse = new TotalApiRegionResponse(
+                List.of(secondResultApiRegionResponse)
+        );
+
+        mockRestServiceServer
+                .expect(requestTo(secondRegionUri))
+                .andRespond(withSuccess(
+                        objectMapper.writeValueAsString(secondOpenApiResultApiRegionResponse),
+                        MediaType.APPLICATION_JSON)
+                );
+
+        final String thirdRegionCode = "11230";
+        final String thirdRegionUri = "https://sgisapi.kostat.go.kr/OpenAPI3/addr/stage.json?" +
+                "accessToken=" +
+                accessToken +
+                "&cd=" +
+                thirdRegionCode;
+        final ResultApiRegionResponse thirdResultApiRegionResponse = new ResultApiRegionResponse(
+                "개포1동",
+                "11230680"
+        );
+        final TotalApiRegionResponse thirdOpenApiResultApiRegionResponse = new TotalApiRegionResponse(
+                List.of(thirdResultApiRegionResponse)
+        );
+
+        mockRestServiceServer
+                .expect(requestTo(thirdRegionUri))
+                .andRespond(withSuccess(objectMapper.writeValueAsString(thirdOpenApiResultApiRegionResponse), MediaType.APPLICATION_JSON));
```

Comment:
```suggestion
.andRespond(withSuccess(
    objectMapper.writeValueAsString(thirdOpenApiResultApiRegionResponse),
    MediaType.APPLICATION_JSON
));
```

개행을 넣으면 더 좋을 것 같습니다!

Reply:
감사합니다..
개행 어렵네요